datacats/cli/profile.py

```diff
@@ -15,31 +17,28 @@ def get_working_profile(environment):
     user interactively.
     """
     profile = UserProfile()
+    if not (profile.ssh_private_key is None) and \
+        exists(profile.ssh_private_key) and \
+        exists(profile.ssh_public_key):
+        key_recognized = profile.test_ssh_key(environment)
+        if key_recognized:
+            # we sucessfully loaded the ssh key and tested it by connecting to the server, so all is good and
+            # we have got ourselves a working profile
+            return profile
+
+        user_error_message = """Your ssh key (which is an equivalent of your password so that datacats.io could recognize you) does not
+        does not seem to be recognized by the datacats.io server.\n \n
+        Most likely it is because you need to go to www.datacats.com/account/key and add the following public key: \n \n {public_key} \n \n
+        to your profile so that datacat's server could recognize you. So maybe try that?
+        If the problem persists, please contact the developer team.""".format(public_key=profile.read_public_key())
```

Comment:
These messages include lots of white-space on the lines following the first. They also go past 90 or so characters making them wrap in common editor configurations. Also "does not" is repeated. Try formatting like this instead:

```python
user_error_message = ("Your ssh key (which is an equivalent of your password so that "
    "datacats.io could recognize you) does not seem to be recognized by the datacats.io server.\n\n"
    "Most likely it is because you need to go to www.datacats.com/account/key and add the "
...
```
```